src/integrations/react.js

```diff
@@ -39,7 +39,7 @@ export function connect(mapStateToProps, actions) {
 					return this.forceUpdate();
 				}
 			};
-			this.componentWillReceiveProps = p => {
+			this.UNSAFE_componentWillReceiveProps = p => {
```

Comment:
It might be possible to change this to:

```js
this.getDerivedStateFromProps = p => {
  props = p;
}
```

Reply:
@developit unfortunately `getDerivedStateFromProps` method must be static.